app/src/components/LoadScreen.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+// passing in
+
+const LoadScreen = ({ graphInfo }) => {
+  // const [graphInfo, setGraphInfo] = React.useState([]);
+
+  // const getGraphInfo = async () => {
+  //   try {
+  //     const response = async () => setGraphInfo(await apiClient.getGraphs());
+  //     const jsonData = await response.json();
+  //     setGraphInfo(jsonData);
+  //     console.log(jsonData);
+  //   } catch (error) {
+  //     console.error(error.message);
+  //   }
+  // };
+
+  // React.useEffect(() => {
+  //   getGraphInfo();
+  // }, []);
+
+  return (
+    <section>
+      <h2>Load Screen</h2>
+      <table>
+        <thead>
+          <tr>
+            <th>id</th>
+            <th>Graph Name</th>
+            {/* <th>User</th> */}
+            {/* <th>Time Saved</th> */}
+          </tr>
+        </thead>
+        <tbody>
+          {graphInfo.map((info) => (
```

Comment:
This seems to work fine although I'm confused as to what info is populating. Seems like a WIP?

Reply:
Yes, I intend these to link to an id param to load data from a single row, but that doesn’t work yet